From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: grayr0ot <thisis41@protonmail.com>
Date: Sun, 13 Aug 2023 12:51:53 +0200
Subject: [PATCH] 	modified:  
 src/main/java/net/minecraft/world/level/block/MushroomBlock.java


diff --git a/src/main/java/net/minecraft/world/level/block/MushroomBlock.java b/src/main/java/net/minecraft/world/level/block/MushroomBlock.java
index ed9ee4d01c69333557824e9f4448a7eaeaa46611..746d8001ada3db7417a426e0b70df486bfbb9477 100644
--- a/src/main/java/net/minecraft/world/level/block/MushroomBlock.java
+++ b/src/main/java/net/minecraft/world/level/block/MushroomBlock.java
@@ -40,12 +40,13 @@ public class MushroomBlock extends BushBlock implements BonemealableBlock {
     @Override
     public void randomTick(BlockState state, ServerLevel world, BlockPos pos, RandomSource random) {
         if (random.nextFloat() < (world.spigotConfig.mushroomModifier / (100.0f * 25))) { // Spigot - SPIGOT-7159: Better modifier resolution
+            int i = 5;
             // Paper start - Increase vanilla mushroom spread
-            int mushroomSpreadLimit = world.paperConfig().environment.mushroom.spreadLimit.or(5);
+            int mushroomSpreadLimit = world.paperConfig().environment.mushroom.spreadLimit.or(i);
             if(world.paperConfig().environment.mushroom.disableLimit.or(false)) {
                 mushroomSpreadLimit = 9 * 9 * 3; // Paper - 9x9x3 Vanilla SHAPE sum
             }
-            int i = mushroomSpreadLimit;
+            i = mushroomSpreadLimit;
             // Paper end
 
             boolean flag = true;
